signalfx-smart-gateway/README.md

```diff
@@ -11,7 +11,11 @@ The SignalFx Smart Gateway observes every transaction across distributed service
 
 ### INSTALLATION
 
-The SignalFx Smart Gateway is available as a single, statically-linked binary. From within the SignalFx UI, the latest version can be downloaded <a target="_blank" href="/#/smart-gateway/download/v2.1.4">here</a> from SignalFx. Alternatively, you can download a specific version of the SignalFx Smart Gateway from the command line using `curl`:
+Note: This content appears on a SignalFx documentation page and on the Setup tab of the Smart Gateway tile in the SignalFx UI. The following link to download the current version works only if you are viewing these instructions on the Setup tab.  
```

Comment:
While my suggestion below is wordier, I think it helps to clearly state the need to access the UI while also mentioning the nuances of viewing content from the doc site versus the UI. Just a different angle to consider. Any thoughts, @barbara-sfx ?

Also, is the tile named "SignalFx Gateway" or "SignalFx Smart Gateway?" I only see "SignalFx Gateway" in the UI; I don't see "SignalFx Smart Gateway" in the UI.

===

The SignalFx Smart Gateway is available as a single, statically linked binary. 

**If you are viewing this content from the SignalFx UI**, then you can download the latest version <link> here.
**If you are viewing this content from the documentation site**, then you must access the SignalFx UI to view and access the above-mentioned link. (Please note, while the link is visible in the documentation site, the link is only functional from the SignalFx UI.) 

1. In the SignalFx UI, click **Integrations** on the navigation bar. 
2. On the SignalFx Integrations page, click SignalFx Smart Gateway. 
3. Click **Setup**, and then locate and click the above-mentioned link.

Alternatively, you can also download a specific version of the SignalFx Smart Gateway from the command line using `curl`:

Reply:
Good suggestion, but TMI given that all that info will also show up in the tile.